pages/index.rst

```diff
@@ -54,62 +54,111 @@ at our `support email`_.
+=======
+* Download the Miniconda installer for Python 3.X `here <http://conda.pydata.org/miniconda.html>`_.
+  **Windows 32-bit machines are NOT supported by most packages and cannot be used.**
+* After downloading the installer, open it and click through the graphical install utility.
+  Accept all of the default installation settings.
+* You should now have a program called “Anaconda Prompt” installed. Open it
+  (this will be your Python command prompt).
+
+Mac/Linux
+=========
+* After downloading the bash installer, open a command prompt (terminal program on the Mac).
+* Change the directory at the terminal to wherever the installer was downloaded. On most systems,
+  this will default to the downloads directory in your user account. If that’s the case, :code:`cd ~/Downloads`
+  will get you there, or replace the path with wherever you saved the file.
+* Run the installer script by typing :code:`bash Miniconda3-latest-MacOSX-x86_64.sh`.
+  Note: Your file name may be different depending upon your operating system! replace Miniconda3-latest-MacOSX-x86_64.sh
+  with whatever the name of the file you downloaded was.
+* Accept the defaults.
+* After the installer has completed completely close and restart your terminal program (this sources the newly modified path).
+* If bash isn't your default shell, switch to it by running the command :code:`bash`.
+* Verify that your install is working by running :code:`conda --version`.
+  You should see a response like conda 4.8.0 or similar (though yours may be a slightly different version number).
+
+*********************
+Setup the Environment
+*********************
+
+Environments are great ways to isolate your sandbox to work in of a given Python version, packages, etc. You'll
+learn more about them later, but we'll need to create one that contains all of the useful packages used in these
+materials. If you can't wait to learn more about environments, checkout this
+`MetPy Monday video <https://www.youtube.com/watch?v=15DNH25UCi0>`_ on them.
+
+* Open a terminal window (Anaconda Prompt if you're on Windows).
+* Download the environment.yml file that tells your system what should be in the environment. Remember where you
+  download this file! Most systems go in ~/Downloads by default which is fine. Right click and select "save" on
+  `this link <https://raw.githubusercontent.com/Unidata/python-workshop/master/environment.yml>`_ to download the file.
+* In the terminal, navigate to wherever this file saved, probably :code:`cd ~/Downloads` will get you there.
+* Run the command :code:`conda env create` and wait for the installation to finish.
+* Run the command :code:`conda activate training` to activate the unidata environment and verify that everything is ready.
+* For an in-depth tutorial on conda and environments, check out this
+  `Carpentry-style tutorial <https://kaust-vislab.github.io/introduction-to-conda-for-data-scientists/>`_.
+
+*************************
+Download Course Materials
```

Comment:
Is this really a good idea given that this downloads the entire repo and not just the notebooks? It's certainly not a step we want people to do for the AMS course.

Reply:
I just kept the git clone instructions and added the zip file download. Maybe we have a page for the AMS course that links to the package we're making? The ZIP is sitting at about 50 MB currently.

Reply:
Fair enough.